src/style.css

```
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+  min-width: 300px;
+  flex-basis: 100%;
+  flex-grow: 1;
```

You may use a shorthand syntax
```suggestion
  flex: 1 1 100%;
```